plugins/project-manager/skills/update/SKILL.md

```
@@ -0,0 +1,252 @@
+```
+
+**Pinging stale assignees:**
+```bash
+gh issue comment ISSUE_NUMBER --body "@username — This issue hasn't been updated in [X] days. Are you still working on it? If not, I can unassign so someone else can pick it up."
+```
+
+### Step 10: Print Summary
+
+After all actions complete, print a final summary:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)